### PR TITLE
[WIP] browser: Update UI with new components and elements

### PR DESCRIPTION
**Pull request by rushenn**
## Description
New UI for browser
ui

**Todo**
- Image preview
- Mobile support (Responsive)
- IE testing

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by vadmeste** [comment]
:1st_place_medal:

**Review by harshavardhana — APPROVED** [review]

**Comment by krishnasrinivas** [comment]
@harshavardhana i could send a separate fix for `the svgs are returning index.html` problem.

**Comment by harshavardhana** [comment]
> @harshavardhana i could send a separate fix for the svgs are returning index.html problem.

Rushen already fixed it.. i tested can you test on your end?

**Comment by krishnasrinivas** [comment]
> Rushen already fixed it.. i tested can you test on your end?

ah, will do.

File: README.md [hunk 1]
```diff
@@ -77,9 +77,7 @@ You can run Minio on FreeBSD with FreeNAS storage-backend - see [here](https://d
 Source installation is only intended for developers and advanced users. If you do not have a working Golang environment, please follow [How to install Golang](https://docs.minio.io/docs/how-to-install-golang).
```
Comment (harshavardhana) [inline README.md]: This change is unrelated to this can you rebase properly locally @rushenn ?

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by rushenn** [comment]
@harshavardhana Rebased, please check.

**Review by harshavardhana — APPROVED** [review]

**Comment by krishnasrinivas** [comment]
path


Previously `>` was `/` which looks more intuitive as a path separator.

i.e the path would look as `test / 1 / 2 / 3`

**Comment by krishnasrinivas** [comment]
click-outside1

Here it is not obvious that you have to click somewhere outside to remove the "share-url" and "delete" buttons from the view.

Older style:


click-outside2

Here it is obvious that you have to click outside .... as the "share-url" and "delete" buttons looks overlaid on a background.

**Comment by krishnasrinivas** [comment]
create

On mouse-over it should show a tool tip like how it was previously. Right now the icons are not clear on its purpose.

**Comment by krishnasrinivas** [comment]
policy

Previously we had three dots and it was intuitive that you had to click it for more options. Now it is not intuitive that you have to click the circle icon (marked in the image)

**Comment by krishnasrinivas** [comment]
bucketlist

Will it be better to darken the background of the bucket selected as well?

**Comment by krishnasrinivas** [comment]
miniobrowser1
miniobrowser2


does the lower case look better?

**Comment by krishnasrinivas** [comment]
topicons

Needs tooltip

**Comment by krishnasrinivas** [comment]
select-file

It will be better to change the file-select icon to circle when the mouse is moved over the icon, now it changes to circle if the mouse is anywhere on the row.

**Comment by krishnasrinivas** [comment]
signout

you can use the `sign-out` icon instead of `power-off` icon

**Comment by rushenn** [comment]
@krishnasrinivas Thank you, I will make sure to include these fixes as part of my next push.

**Comment by rushenn** [comment]
> It will be better to change the file-select icon to circle when the mouse is moved over the icon, now it changes to circle if the mouse is anywhere on the row.

If the checkbox toggle is set to the icon only, it won't be noticed much. Since the checkboxes are hidden by default, i think it's better to keep the toggle set to the entire row.

**Comment by deekoder** [comment]
Update PR with responsiveness changes.

**Pull request closed without merging**